packages/react/src/Pillar/Pillar.module.css

```diff
@@ -15,6 +15,17 @@
   text-align: left;
 }
 
+.Pillar__image {
+  margin-bottom: var(--base-size-28);
+  border-radius: var(--brand-borderRadius-medium);
```

Comment:
All border-radius for cards, bento's etc are supposedly increasing to xlarge. Can we apply that now? Maybe also sneak in the same change into Card too?

Reply:
I prefer that we handle this in a dedicated PR to ensure all components that need updating are addressed.

Reply:
Okay, mentioning as I can see [you've made a padding update to the Card in this PR](https://github.com/primer/brand/pull/862/files#diff-666b3aa4d4d292698c805bf515a4cd8ff79c3d7f4db003e28aaf4efe81bde8c8R103) already, so felt okay to do it in this PR too? Fine if you want to defer it also.